src/resources/Microsoft.Authorization/policyDefinitions/Deploy-Diagnostics-PostgreSQL.json

```diff
@@ -9,7 +9,7 @@
     "displayName": "Deploy Diagnostic Settings for Database for PostgreSQL to Log Analytics workspace",
     "description": "Deploys the diagnostic settings for Database for PostgreSQL to stream to a Log Analytics workspace when any Database for PostgreSQL which is missing this diagnostic settings is created or updated. The Policy will set the diagnostic with all metrics and category enabled",
     "metadata": {
-      "version": "1.1.0",
+      "version": "1.2.0",
```

Comment:
Can we make this a major update please as per https://aka.ms/alz/policies#versioning

Reply:
Version updated